hik_camera:
  ros__parameters:
    camera_sn: "00F26632041"
    exposure_time: 5000.0
    gain: 15.0
    frame_rate: 30.0 
    pixel_format: "BGR8"
    source_type: "file"  
    video_file_path: "/home/coordsys/zbx/MVSROS_project/MVS_ROS2/hik_camera/resource/blue.mp4"    